features/support/ui/pages/regional_strategy.rb

```diff
@@ -0,0 +1,17 @@
+module UI
+  module Pages
+    class Countries < SitePrism::Section
+      element :title, 'h2'
+      element :content, 'p'
+      elements :links, 'a'
+    end
+
+    class RegionalStrategy < ::UI::Pages::Article
+      set_url '/en/regional_strategies/{/slug}'
+
+      element :main_description, '.hero'
+      element :latest_news, '.latest-news'
+      sections :countries, Countries, '.list--countries--item'
```

Comment:
I'm not sure this is related to the work in this PR.... but I don't think we should be having a class named 
`'.list--countries--item'` .. it's not following the BEM conventions that we use here at MAS

Reply:
good spot @willhall88 ♻️